openrtb_ext/bidders_test.go

```diff
@@ -153,4 +156,123 @@ func TestSetAliasBidderName(t *testing.T) {
 	aliasBidderToParentBidder = map[BidderName]BidderName{}
 }
 
+type mockParamsHelper struct {
+	fs              fstest.MapFS
+	absFilePath     string
+	absPathErr      error
+	schemaLoaderErr error
+	readFileErr     error
+}
+
+func (m mockParamsHelper) readDir(name string) ([]os.DirEntry, error) {
```

Comment:
`(m *mockParamsHelper) ` should we have pointer receiver here? to ensure same instance of `mockParamsHelper` is being used. Also should store input parms in `mockParamsHelper`. These stored values can be asserted latter to make sure expected params are passed as input

Reply:
> `(m *mockParamsHelper) ` should we have pointer receiver here and on other? to ensure same instance of `mockParamsHelper` is being used.
https://github.com/prebid/prebid-server/pull/3048/commits/41cc3fa63f7da536bebef841f11fafc13c539b05

> Also should store input parms in `mockParamsHelper`. These stored values can be asserted latter to make sure expected params are passed as input

I am still over the edge if we would need to assert the inputs of each mocked function. Given that we have mocked the callers and we are sure that no actual functions are being called since that will error out and won't yield the expected output that is already asserted. Honestly, our focus should be on asserting the output and in making sure that expected file schema is being returned or not which is what the test case is doing. I don't think of any compelling reasons to assert the mocked function inputs with the expected value since that is implicitly being tested.